internal/host/validator.go

```diff
@@ -845,13 +860,14 @@ func allImagesValid(imageStatuses common.ImageStatuses) bool {
 	return true
 }
 
-/*
-   This is a pre-install validation that checks that the boot device was either not tested for sufficient disk speed
-   or the disk speed check has been successful.  Since disk speed test is performed after installation has started,
-   in order to have result for such test, the result has to be from a previous installation attempt.
-   Since all pre-install validations have to pass before starting installation, it is mandatory that in case installation
-   on the current boot device has not been attempted yet, this validation must pass.
-*/
+// This is a pre-install validation that checks that the boot device was either
```

Comment:
Recent gopls versions are opinionated about how this should be indented due to changes to the go doc format, I changed it to regular single-line comments to avoid this mismatch